Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -48,8 +48,10 @@ public class PressableButton : MonoBehaviour, IMixedRealityTouchHandler
 
         [Header("Position markers")]
```

Comment:
I believe having the Tooltip attribute here will cause a compile error - "Duplicate tooltip attribute"